### ISSUE TITLE
Setting default value as empty string for datetimepicker 

### ISSUE THREAD

**Issue by muratcatal** [dmtrKovalenko/material-ui-pickers#198]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers |  1.0.0-beta.13.1       |
| material-ui         |    1.0.0-beta.30    |
| React               |    16.2.0     |
| Browser             |   chrome 63.0.3239.132      |
| Platform            |  windows       |

I want to give empty value as default value to datetimepicker. To do this, i tried to set its value to null/undefined/empty string as first value, however it shows invalid date message. How can i set an initial value as empty string? I couldn't see in docs or in examples.

 ```
the alarmStartTimeKeyword1's value is set to null / undefined or empty string in constructor.

<DateTimePicker
	disablePast
	placeholder="GG.AA.YYYY SS:DD"
	className = {classes.time}
	keyboard = {true}
	keyboardIcon = {<Icon> <InsertInvitationIcon/> </Icon>}
	invalidDateMessage = {Translator.t('Invalid Date Format')}
	animateYearScrolling={true}
	ampm={false}
	returnMoment={false}
	format="DD.MM.YYYY HH:mm"
	value = {this.state.alarmStartTimeKeyword1}
	onChange = {this.handleDate('alarmStartTimeKeyword1')}
	timeIcon = { <Icon> <AlarmIcon/> </Icon>}
	dateRangeIcon = { <Icon> <DateRangeIcon/> </Icon>}
	leftArrowIcon = { <Icon> <ChevronLeftIcon/> </Icon>}
	rightArrowIcon = { <Icon> <ChevronRightIcon/> </Icon>}
	okLabel = {Translator.t('ok')}
	cancelLabel = {Translator.t('cancel')}
/>
```

![untitled](https://user-images.githubusercontent.com/5106209/35497467-4bbd67d4-04db-11e8-95ff-9623348eb867.png)

## Expected behavior
Not to give invalid date message for default value if its empty

**Comment by santosguilherme** [comment]
+1

**Comment by cherniavskii** [comment]
From a fast look it's a bug.
Maybe introduced in #188

**Comment by muratcatal** [comment]
@sakulstra how can i set datetimepicker's value empty for default value? think that i have a page which has some filter parameters and one of the filter parameter comes from that datetimepicker. and because of i cant set its value to empty in initial state, filtering could not work as expected. 
so if its not a bug which an empty value could not be set to datetimepicker as an initial state, and it should be something that must be discussed, there must be a best-practice to handle that issue which i faced with during development.

**Comment by cherniavskii** [comment]
Just submitted a fix for that - #199
